index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <header>
+      <nav id="navbar">
+        <ul class="navbar-nav ml-auto">
+          <li class="nav-item">
+            <a class="nav-link" href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#projects">Project</a>
+          </li>
+          <li class="nav-item">
+            <a class="nav-link" href="#contact">Contact</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <!-- <h1>Hello World, we are software developers!</h1> -->
+    <script src="js/index.js"></script>
+    <section id="welcome-section">
+      <div>
+        <h1>Hey we are Team Elio and Amrendra</h1>
+        <p>We are Web Developer</p>
+      </div>
+    </section>
+    <section id="projects">
+      <div>
+        <h1 class="project-tile">Projects</h1>
+        <p>List of projects</p>
```

Comment:
h1 tag usually comes to denote the Head section of the page. It is nice to use other h2,h3,h4 using in between of pages.

Reply:
- [ ] Good job on this review, but apart from explaining the reasons for the requested change. You can request for this change using polite terms like please and kindly